src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
```

Comment:
next time I'll reject
```suggestion
    <form 
      action="/"
      method="post"
      data-qa="big"
      class="form"
    >
```